napari/components/viewer_model.py

```diff
@@ -470,8 +470,8 @@ def _subplot(self, layer, position):
             Size of the grid that is being used.
         """
         extent = self._sliced_extent_world
-        scene_size = extent[1] - extent[0]
-        translate_2d = np.multiply(scene_size[-2:], position)
+        scene_shift = extent[1] - extent[0] + 1
```

Comment:
I changed this to make it so next image starts one pixel after the previous image ends. It looks like the same behavior on master and the test passes. Is this right, @sofroniewn ?

Reply:
Yes, I think this makes sense, thanks for that fix!!